lib/async_hooks.js

```diff
@@ -60,19 +60,11 @@ async_wrap.setupHooks({ init,
                         destroy: emitDestroyN });
 
 // Used to fatally abort the process if a callback throws.
-function fatalError(e) {
```

Comment:
Why is everyone trying to kill this :fearful:
I like it as a way to bail from a callback of a Promise.catch
Ref: https://github.com/nodejs/node/pull/13839#discussion_r124936232